### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by sudaisasif**
Fixing some grammatical mistakes, spelling errors & sentence structures.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,26 +35,26 @@ Listing a few git resources here for everyone.
...
  ### Introduction of Git Version Control System and Getting Started Guide
- The starting point was [Github in 15 Minuets](http://www.try.github.com) , but Mr. Ahmad Awais explained basic Git command and working of the git version control by relating them with real world examples and comparing it with the problem scenarios faced by the people who don't know or don't want to use the version control system. Following is the summary:
+ The starting point was [Github in 15 Minutes](http://www.try.github.com) , but Mr. Ahmad Awais explained basic Git commands and working of the git version control by relating them with real world examples and comparing it with the problem scenarios faced by the people who don't know or don't want to use a version control system. Following is the summary:
  * Initializing a Git Directory
- * Add something in that repository
+ * Adding something in that repository
```
Comment (ahmadawais) [inline README.md]: `Add` is better than `Adding` revert that back. Git commits should be written as you are ordering Git to do something. Add this, Do that, Improve them, etc. Adding is you doing something and is not good.

File: README.md [hunk 2]
```diff
@@ -35,26 +35,26 @@ Listing a few git resources here for everyone.
...
  * Commit changes with a Message
  * Branches in git
  * Push from local Master branch to remote Master branch
  * Pull from remote Master branch to Local Master branch
  * Forking an open source project 
  * Cloning Forked Project
- * Creating a Pull Request for the open source Project
- * And some Professionalism in doing above activities (i.e how should be Commit Message)
+ * Creating a Pull Request for an open source Project
+ * And some Professionalism in doing above activities (i.e how should we commit a message)
```
Comment (ahmadawais) [inline README.md]: You don't commit a message. There's a Commit Message alongside your commit. `how should be your Commit Messages` would be the right thing here.

**Comment by ahmadawais** [comment]
I'm closing this one since now there are conflicts. Create a new PR!  @sudaisasif

**Comment by sudaisasif** [comment]
Sure. :)

**Pull request closed without merging**